tests/integ/test_simplifier_suite.py

```diff
@@ -57,6 +57,21 @@ def skip(pytestconfig):
         )
 
 
+paramList = [True, False]
+
+
+@pytest.fixture(params=paramList)
+def setup_reduce_cast(request, session):
```

Comment:
Can this be done using pytest.parameterize?

Reply:
yes, but i will need to add a finally to reset the parameter back at the end of each tests, this setup function captures all steps, easier for reuse the code, would prefer to use the fixture here since more than one test point is involved